harmony/harmonydb/sql/20240621-harmony-mch-task-list.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE harmony_task ADD COLUMN created_time TIMESTAMP NOT NULL DEFAULT current_timestamp;
+CREATE TABLE harmony_task_bid (
+    id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
Should we create a constraint that this ID will only exists if it exists in harmony_task table?